### PR TITLE
Translation - Improve German translation

### PR DESCRIPTION
**Pull request by OverlordZorn**
🇩🇪
got *slightly* sidetracked while working on the german guide 😅

### CODE REVIEW

**Comment by MissHeda** [comment]
Why did you open a new translation PR when the is currently still one pending for German translation?
Won't this one interfere with the one that is already open?

**Comment by OverlordZorn** [comment]
> Why did you open a new translation PR when the is currently still one pending for German translation? Won't this one interfere with the one that is already open?

thats not merged yet?

also, `Atemwegsverlegungswahrscheinlichkeit` triggered me

I dont mind reiterating over my PR once that one gets resolved.

**Comment by OlafZigel** [comment]
> > Why did you open a new translation PR when the is currently still one pending for German translation? Won't this one interfere with the one that is already open?
> 
> thats not merged yet?
> 
> also, `Atemwegsverlegungswahrscheinlichkeit` triggered me
> 
> I dont mind reiterating over my PR once that one gets resolved.

If you want we could update my PR with the problems you fixed :D
Because I can't understand what the problem is

**Comment by OverlordZorn** [comment]
I have merged @OlafZigel 's PR #493 into this PR, picking the best of both worlds.
I have merged tomcat-dev into this PR beforehand, meaning any accidental changes of the Japanese Lines should be reverted.

Afterwards, I double checked the files with tabler, no errors occour, meaning there are no duplicate entries left.


```
@MiszczuZPolski 
I dont know whats going on here:
![grafik](https://github.com/KAT-Advanced-Medical/KAM/assets/56258612/3ac5e61b-8641-409c-81a4-8ae415588098)

My local version seems fine
![grafik](https://github.com/KAT-Advanced-Medical/KAM/assets/56258612/00dd2188-e548-4d49-8c2a-da6597eddeb9)

dev-tomcat version:
https://github.com/KAT-Advanced-Medical/KAM/blob/b244f4360ac88d2631d88abfb81d6d2ad788f6a5/addons/circulation/stringtable.xml#L2047-L2051
```
Nvm, had an old version of KAM on my fork. now its not complaining anymore.

File: addons/pharma/stringtable.xml [hunk 1]
```diff
@@ -2754,7 +2756,7 @@
             <Italian>Possibilità di ostruzione della IV senza prima aver svolto uno sciacquo salino</Italian>
             <French>Chance de blocage des IV dû à une administration intra-veineuse de médicaments sans rinçage à la solution saline</French>
             <Korean>식염수 주입 없이 약물 투여로 인한 IV 막힘 확률</Korean>
-            <German>Möglichkeit einer Verstopfung der IV durch Medikamentengabe ohne Kochsalzspülung</German>
+            <German>Möglichkeit einer Okklusion der IV durch Medikamentengabe ohne Kochsalzspülung</German>
```
Comment (OlafZigel) [inline addons/pharma/stringtable.xml]: Es ist eine Verstopfung des IV. Also würde ich das auch dabei belassen :D

Comment (OverlordZorn) [reply addons/pharma/stringtable.xml]: is das nich von dir?

Comment (OlafZigel) [reply addons/pharma/stringtable.xml]: Kann sein.. bitte ändere das um :D

**Comment by OverlordZorn** [comment]
resolve #493

**Pull request merged**